src/sentry/explore/models.py

```diff
@@ -0,0 +1,83 @@
+from __future__ import annotations
+
+from typing import Any
+
+from django.db import models, router, transaction
+from django.utils import timezone
+
+from sentry.backup.scopes import RelocationScope
+from sentry.db.models import FlexibleForeignKey, Model, region_silo_model, sane_repr
+from sentry.db.models.fields import JSONField
+from sentry.db.models.fields.bounded import BoundedBigIntegerField, BoundedPositiveIntegerField
+from sentry.db.models.fields.hybrid_cloud_foreign_key import HybridCloudForeignKey
+from sentry.models.dashboard_widget import TypesClass
+
+
+class ExploreSavedQueryTypes(TypesClass):
+    SPANS = 0
+
+    TYPES = [
+        (SPANS, "spans"),
+    ]
```

Comment:
Can we add a `logs` type here? It will be unused anyway until we write to it.

Reply:
added `ourlogs`